bottom_sheet/lib/main.dart

```diff
@@ -0,0 +1,154 @@
+          children: <Widget>[
+            new RaisedButton(
+              child: Text("Bottom Sheet"),
+              onPressed: () {
+                _bottomSheetMore(context);
+              },
+            )
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+void _bottomSheetMore(context) {
+  showModalBottomSheet(
+    context: context,
+    builder: (builder) {
+      return new Container(
+        padding: EdgeInsets.only(
+          left: 5.0,
+          right: 5.0,
+          top: 5.0,
+          bottom: 5.0,
+        ),
+        decoration: new BoxDecoration(
+            color: Colors.white,
+            borderRadius: new BorderRadius.only(
+                topLeft: const Radius.circular(10.0),
+                topRight: const Radius.circular(10.0))),
+        child: new Wrap(
+          children: <Widget>[
+            new ListTile(
+              title: const Text(
+                'More',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+            ),
+            new ListTile(
+              leading: new Container(
+                width: 4.0,
+                child: Icon(
+                  Icons.favorite,
+                  color: Colors.pink,
+                  size: 24.0,
+                ),
+              ),
+              title: const Text(
+                'Favourites',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+            ),
+            new ListTile(
+              leading: new Container(
+                width: 4.0,
+                child: Icon(
+                  Icons.settings,
+                  color: Colors.black,
+                  size: 24.0,
+                ),
+              ),
+              title: const Text(
+                'Settings',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+            ),
+            new ListTile(
+              leading: new Container(
+                width: 4.0,
+                child: Icon(
+                  Icons.account_box,
+                  color: Colors.blue,
+                  size: 24.0,
+                ),
+              ),
+              title: const Text(
+                'Profile',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+            ),
+            new Divider(
+              height: 10.0,
+            ),
+            new ListTile(
+              title: const Text(
+                'Logout',
+                style: TextStyle(
+                  fontSize: 14.0,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+              onTap: () async {
+                // Add Here
```

Comment:
Noting happens when you click on the items in Bottom Sheet. Ideally it should do something, for the least it should dismiss the Bottom Sheet.